x/staking/keeper/delegation.go

```diff
@@ -253,22 +253,6 @@ func (k Keeper) IterateDelegatorDelegations(ctx sdk.Context, delegator sdk.AccAd
 	}
 }
 
-// IterateDelegatorRedelegations iterates through one delegator's redelegations.
-func (k Keeper) IterateDelegatorRedelegations(ctx sdk.Context, delegator sdk.AccAddress, cb func(red types.Redelegation) (stop bool)) {
```

Comment:
We should not delete that, this is used by apps: https://github.com/search?q=%22.IterateDelegatorRedelegations%28%22+lang%3AGo&type=code